server/vulnerabilities/oval/oval_platform.go

```diff
@@ -0,0 +1,41 @@
+package oval
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+)
+
+// Combines the host platform and os version into 'platform-major version' string,
+// ex: ('ubuntu', 'Ubuntu 20.4.0') => 'ubuntu-20'
+type Platform string
+
+const OvalFilePrefix = "fleet_oval"
+
+// Returns the major version of an 'os_version',
+// ex: 'Ubuntu 20.4.0' => '20'
+func getMajorRelease(osVersion string) string {
+	re := regexp.MustCompile(` (?P<major>\d+)\.?(\d+)?\.?(\*|\d+)?$`)
+	m := re.FindStringSubmatch(osVersion)
+	idx := re.SubexpIndex("major")
+
+	if idx < len(m) {
+		return m[idx]
+	}
+	return ""
+}
+
+func NewPlatform(hostPlatform, hostOsVersion string) Platform {
```

Comment:
Document that `hostPlatform` and `hostOsVersion` are values coming from `osquery` (correct?)

Reply:
They actually come from here https://github.com/fleetdm/fleet/blob/main/server/fleet/hosts.go#L317 - which is based on the `hosts` table.